src/lib/manifestationFactoryUtils.js

```diff
@@ -1,22 +1,47 @@
 import { chain, isEqual, uniqWith, upperFirst } from "lodash";
 import { getCoverImage } from "@/components/utils/getCoverImage";
 
+/**
+ * Format to array from url
```

Comment:
et ekempel i doc ville være godt - (fisk/hest) -> [fisk,hest] - så behøver man ikke at læse koden for at forstå det